### PR TITLE
Update Guzzle version to 7 and Guzzle-Services to 1.2

### PR DESCRIPTION
**Pull request by MohammadWaleed**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
         "chat": "https://www.hipchat.com/gIdidQscL"
     },
     "require": {
-        "php":           ">=5.6.0",
-        "ext-openssl":    "*",
-        "guzzlehttp/guzzle": "~6.0",
-        "guzzlehttp/guzzle-services": "~1.0",
+        "php": ">=7.0",
+        "ext-openssl": "*",
+        "guzzlehttp/guzzle": "~7.0",
```
Comment (stof) [inline composer.json:16]: I'd rather keep `^6.0 || ^7.0` (and 1.0 as min version for `guzzlehttp/guzzle-services`, to stay compatible with projects which are still using Guzzle 6.

**Review by stof — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (stof) [reply composer.json:16]: also not changing the min PHP version then

Comment (MohammadWaleed) [reply composer.json]: I updated the versions constraints to support legacy projects

File: CHANGE.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+VERSION 2.8.0
+-------------
+* Bumbed Guzzle version to 7
+* Bumbed Guzzle Services version to 1.2
```
Comment (niektenhoopen) [inline CHANGE.md:7]:
```suggestion
* Bumped Guzzle Services version to 1.2
```

File: CHANGE.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+VERSION 2.8.0
+-------------
+* Bumbed Guzzle version to 7
```
Comment (niektenhoopen) [inline CHANGE.md:6]:
```suggestion
* Bumped Guzzle version to 7
```

File: composer.json [hunk 4]
```diff
@@ -11,10 +11,10 @@
...
     },
     "require": {
-        "php":           ">=5.6.0",
-        "ext-openssl":    "*",
-        "guzzlehttp/guzzle": "~6.0",
-        "guzzlehttp/guzzle-services": "~1.0",
+        "php": ">=5.6.0",
+        "ext-openssl": "*",
+        "guzzlehttp/guzzle": "^6.0 || ^7.0",
+        "guzzlehttp/guzzle-services": "^1",
```
Comment (niektenhoopen) [inline composer.json:17]:
```suggestion
        "guzzlehttp/guzzle-services": "^1.0",
```

**Review by niektenhoopen — APPROVED** [review]
Tests seem ok. Code style too. I hope @elijahchancey will test it in his project.
Closes #140

**Comment by elijahchancey** [comment]
It works great. Thanks! Please merge when you can.

**Review by stof — APPROVED** [review]

**Comment by stof** [comment]
@piotrek-buchman @kmfk could you please merge this and release it ?
I have been granted maintainer access on the Symfony bundle, but not on the PHP SDK, so I cannot do it.

**Review by kmfk — APPROVED** [review]
👍

**Comment by piotrek-buchman** [comment]
@stof thanks for your PR, this is a great improvement and will be appreciated by our community.

**Comment by stof** [comment]
@piotrek-buchman it is not my PR. Credits go to @MohammadWaleed

**Comment by piotrek-buchman** [comment]
I'm sorry, I clicked on a wrong link. Thanks go to @MohammadWaleed as well as all reviewers!

**Pull request merged**